src/data/get_runway_stats.py

```diff
@@ -21,14 +21,23 @@ class RunwayStats:
     _runways_stats_df: pd.DataFrame = field(init=False)
 
     def __post_init__(self, runways: pd.DataFrame) -> None:
-        self._runways_stats_df = runways.map(self.count_runways)
-        self._runways_stats_df['total_runway_length'] = runways.map(self.sum_runways_len)
-
-    def count_runways(self, airport_runways: list[RunwayDict]) -> int | None:
-        return len(airport_runways) or None
-
-    def sum_runways_len(self, airport_runways: list[RunwayDict]) -> int | None:
-        return sum(runway['length_in_ft'] for runway in airport_runways) or None
+        runways_count = list(map(self.count_runways, runways['runways'], runways['iata']))
```

Comment:
Why would a df of runways have an `iata` field?

Reply:
Is this actually an airport rather than just the runways?

Reply:
yeah actaully that is kind of confusing its a really a df of a list of runways at different airports - i'll fix the naming